Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Stage: base image
-FROM node:18.16-bullseye-slim as base
+FROM node:20.8-bullseye-slim as base
```

Comment:
we need to stay on the LTS version so need to be `FROM node:18.18-bullseye-slim as base`

Reply:
done - though this comes from a PR kicked off by renovate-bot https://github.com/ministryofjustice/hmpps-authorization/pull/8